course/meetings/eeb-marty-asher-2017-03-30.md

```diff
@@ -0,0 +1,82 @@
+---
+contact: Asher Cutter, Martin Krkosek
+department: Ecology and Evolutinoary Biology
+uoftcoders: Madeleine, Luke, Lina, Joel, Lindsay, Elliot
```

Comment:
@QuLogic The reason I keep misspelling your name is because my neighbour's name is spelled with only one 't', so I never react to it looking incorrect...